yt/visualization/fixed_resolution_filters.py

```diff
@@ -7,9 +7,11 @@
 
 def apply_filter(f):
```

Comment:
Should this go to the house of `commons`?

Reply:
It's in line with the rest of the decorators I've moved there, but my original motivation was to make them easily importable from other modules, which I ended up not doing, so it would feel equally right to revert that code migration. Whatever you or other reviewers feel is preferable suits me :)